.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: rust
+rust: nightly
```

Comment:
Ah, good call.  I thought I was running 1.0.0, but I guess that I am actually not based on how I compiled the source.